### ISSUE TITLE
The CI always fails on ubuntu-latest for JDK 8 with TransactionBackendHandlerFactoryTest

### ISSUE THREAD

**Issue by terrymanu** [apache/shardingsphere#12729]
```log
Error:  Tests run: 3, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 0.21 s <<< FAILURE! - in org.apache.shardingsphere.proxy.backend.text.transaction.TransactionBackendHandlerFactoryTest
Error:  assertTransactionBackendHandlerReturnedWhenTCLStatementInstanceOfCommitStatement(org.apache.shardingsphere.proxy.backend.text.transaction.TransactionBackendHandlerFactoryTest)  Time elapsed: 0.083 s  <<< ERROR!
java.lang.NullPointerException
	at org.apache.shardingsphere.proxy.backend.text.transaction.TransactionBackendHandlerFactoryTest.assertTransactionBackendHandlerReturnedWhenTCLStatementInstanceOfCommitStatement(TransactionBackendHandlerFactoryTest.java:49)

Error:  assertTransactionBackendHandlerReturnedWhenTCLStatementInstanceOfRollbackStatement(org.apache.shardingsphere.proxy.backend.text.transaction.TransactionBackendHandlerFactoryTest)  Time elapsed: 0.058 s  <<< ERROR!
java.lang.NullPointerException
	at org.apache.shardingsphere.proxy.backend.text.transaction.TransactionBackendHandlerFactoryTest.assertTransactionBackendHandlerReturnedWhenTCLStatementInstanceOfRollbackStatement(TransactionBackendHandlerFactoryTest.java:61)

[INFO] Running org.apache.shardingsphere.proxy.backend.text.transaction.TransactionBackendHandlerTest
Error:  Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.004 s <<< FAILURE! - in org.apache.shardingsphere.proxy.backend.text.transaction.TransactionBackendHandlerTest
Error:  assertExecute(org.apache.shardingsphere.proxy.backend.text.transaction.TransactionBackendHandlerTest)  Time elapsed: 0 s  <<< ERROR!
java.lang.NullPointerException
	at org.apache.shardingsphere.proxy.backend.text.transaction.TransactionBackendHandlerTest.assertExecute(TransactionBackendHandlerTest.java:40)
```

**Comment by LeeGuoPing** [comment]
May I have a try

**Comment by strongduanmu** [comment]
Fixed by #12755.
